docs/02.API-REFERENCE.md

```diff
@@ -5317,15 +5317,23 @@ jerry_parse_and_save_literals (const jerry_char_t *source_p,
                                bool is_strict,
                                uint32_t *buffer_p,
                                size_t buffer_size,
-                               bool is_c_format);
+                               bool is_c_format,
+                               const char *save_literals_file_name_p,
```

Comment:
JerryScript does not know about the concept of files.

Reply:
What do you mean by this?

Reply:
You cannot use fopen/fclose/etc. functions in JerryScript. Theis provides the most compatibility with low-end systems.